Makefile

```diff
@@ -1,7 +1,11 @@
 _DOCKER_ ?= docker
 
+# Build all microservices
 build_all:  build_base build_base_3.12 build_annotation build_users build_convert build_jobs build_keycloak build_assets build_web  build_processing build_taxonomy clean
 
+# Build only BadgerDoc microservices
+build_badgerdoc: build_base build_base_3.12 build_annotation build_users build_convert build_jobs build_assets build_web  build_processing build_taxonomy clean
```

Comment:
Do we use python base 3.12? Do we need it here at this moment?

Reply:
You're right - we're not using it right now, however, base build already done and I don't know the reason why we shouldn't build it during CI